specification/security/resource-manager/Microsoft.Security/preview/2022-11-20-preview/apiCollections.json

```diff
@@ -0,0 +1,298 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Defender for Cloud",
+    "description": "API spec for Microsoft.Security (Microsoft Defender for Cloud) resource provider",
+    "version": "2022-11-20-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
```

Comment:
The Namespace name needs to be reviewed by the naming council. Also pl indicate the differences between the previously approved PR and the current PR if any other than the change from proxy to extension resource model and the namespace name to make the review go fast.

Reply:
@rkmanda sorry, I'm a bit confused. Which part of this file is the namespace name?

The differences between the previously approved PR (https://github.com/Azure/azure-rest-api-specs-pr/pull/8257) and this one are:
1. apiCollections was a proxy resource in the Microsoft.ApiSecurity RP, but we are changing it to an extension resource in the Microsoft.Security RP. This is because we have discovered issues in our private preview, and the switch to an extension resource not only fixes those issues but also is a better fit for the resource.
2. The Microsoft.Security RP is not an RPaaS managed UserRP (for historical reasons unknown to me), and so the response body for the APIs contains the basic fields that every ARM resource must return, like id, type, name.
3. There is now a LIST API for apiCollections because the build validation mandates it.

Reply:
thanks for the clarifications!